packages/core/test/test-util/test-services/v2/test-service/function-imports.ts

```diff
@@ -217,7 +217,7 @@ export function testFunctionImportSharedEntityReturnTypeCollection(
 ): Omit<
   FunctionImportRequestBuilderV2<
     TestFunctionImportSharedEntityReturnTypeCollectionParameters,
-    never
+    never[]
```

Comment:
Is this on purpose?

Reply:
I think we had a bug in the code missing the `[]` since a collection is returen I think this is correct